src/ingestors/index.ts

```diff
@@ -9,6 +12,7 @@ export type MintIngestionMap = {
 export const ALL_MINT_INGESTORS: MintIngestionMap = {
   'prohibition-daily': new ProhibitionDailyIngestor(),
   fxhash: new FxHashIngestor(),
+  'zora-doodles':new zoraIngestor(),
```

Comment:
this should work for all zora mints, right? Not just Doodles?

Reply:
Yes i just added as for demo, will change it to another name